### PR TITLE
Adds error handling to Login page.

### PR DESCRIPTION
**Pull request by adeelkhan**
This patch handles 400 error for login
page and displays backend error to learner.

VAN-105

Screen Shot 2020-11-02 at 4 27 54 PM

### CODE REVIEW

File: src/logistration/LoginFailure.jsx [hunk 1]
```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from '@edx/frontend-platform/i18n';
+
+import { Alert, Hyperlink } from '@edx/paragon';
+
+const processLink = (link) => {
+  let matches;
+  link.replace(/(.*)<a href=["']([^"']*).*>([^<]+)<\/a>(.*)/g, function () { // eslint-disable-line func-names
```
Comment (adeelehsan) [inline src/logistration/LoginFailure.jsx:9]: Can we move that expression to constant's or at least assign to a variable ?

File: src/logistration/LoginPage.jsx [hunk 2]
```diff
@@ -13,6 +13,26 @@ import { DEFAULT_REDIRECT_URL } from './data/constants';
 import { loginRequestSelector, thirdPartyAuthContextSelector } from './data/selectors';
 import LoginHelpLinks from './LoginHelpLinks';
 import RedirectLogistration from './RedirectLogistration';
+import LoginFailureMessage from './LoginFailure';
+
+
+const LoginRedirect = (props) => {
```
Comment (adeelehsan) [inline src/logistration/LoginPage.jsx:19]: That code is already there. Please rebase.

**Review by adeelehsan — CHANGES_REQUESTED** [review]
looks good only few nit's

File: src/logistration/LoginPage.jsx (hunk 2 above)
Comment (adeelkhan) [reply src/logistration/LoginPage.jsx]: Done!

File: src/logistration/LoginFailure.jsx (hunk 1 above)
Comment (adeelkhan) [reply src/logistration/LoginFailure.jsx:9]: This has single usage, so i kept it like this. Name of the function do explain what the regex is doing.

**Review by adeelehsan — APPROVED** [review]

**Pull request merged**